core/commands/stat.go

```diff
@@ -116,14 +150,14 @@ var statBwCmd = &cmds.Command{
 					totals := nd.Reporter.GetBandwidthTotals()
 					out <- &totals
 				}
-				if !doPoll {
-					return
-				}
-				select {
```

Comment:
but the change is that now if `doPoll` is true, it still exits. It just waits, and then exits